code/wdb/datastripe.q

```diff
@@ -31,9 +31,9 @@ modaccess:{[accesstab]};
     .lg.o[`reload;"Kept ",string[.ds.periodstokeep]," period",$[.ds.periodstokeep>1;"s";""]," of data from : ",", " sv string[t]];
     
     // update the access table on disk
-    atab:get ` sv(.ds.td;.proc.procname;`access);
+    atab:get ` sv(.ds.td;.proc.procname;`$ string .wdb.currentpartition;`access);
```

Comment:
```suggestion
    accesspath: ` sv(.ds.td;.proc.procname;`$ string .wdb.currentpartition;`access)
    atab:get accesspath;
```

Reply:
At that point would it not make more sense to make the accesspath a global

Reply:
Actually what I said wouldn't work as it needs to be within one of the functions called by the wdb in order to get currentpartition